website/organizations/index.html

```diff
@@ -22,14 +22,79 @@ <h3 class="header-section">
+          {{ _('Thunderbird is available in a <a href="{syslang}">variety of languages</a> for all major operating systems (Windows / MacOS / Linux). For multi-lingual environments, you can download language pack add-ons and switch Thunderbird's user interface language on the fly. We offer a number of resources specifically for organizations:')|f(syslang=url('thunderbird.latest.all')) }}
         </p>
+
+        <ul class="font-md tracking-wider leading-loose mb-6 p-links-blue">
+          <li>
+            {{ _('For Windows deployment, you can find the Thunderbird MSI Installer on the <a href="{syslang}">Systems &amp; Languages page</a>.')|f(syslang=url('thunderbird.latest.all')) }}
+          </li>
+          <li>
+            {{ _('For more information about deploying and configuring Thunderbird in business and enterprise environments, please visit our enterprise deployment portal: <a href="{thunderbird_etn}">enterprise.thunderbird.net</a>.')|f(thunderbird_etn=url('thunderbird.enterprise.documentation')) }}
+          </li>
+          <li>
+            {{ _('You can also join the <a href="{tb_list_enterprise}">Thunderbird Enterprise mailing list on Topicbox</a> to connect with other administrators and share advice on Thunderbird deployment (Please note: The <a href="tb_oldlist_enterprise">old tb-enterprise mailing list</a> has been retired).')|f(tb_list_enterprise=url('thunderbird.lists.enterprise'), tb_oldlist_enterprise=url('thunderbird.oldlists.enterprise')) }}
+          </li>
+        </ul>
+
         <p class="font-lg tracking-wider leading-loose mb-10 mt-0 p-links-blue">
-          {{ _('For general help or questions about Thunderbird you can visit <a href="{support_site}">Thunderbird support</a>.')|f(support_site=url('support')) }}
+          {{ _('For general help or questions about Thunderbird, you can visit <a href="{support_site}">Thunderbird support</a>.')|f(support_site=url('support')) }}
         </p>
+
+        <h3 class="subheader-section">
+          {{ _('Benefits of using Thunderbird in an organization') }}
+        </h3>
+        <ul class="font-md tracking-wider leading-loose mb-6 p-links-blue">
+          <li>
+            {{ _('Cross-platform desktop email client packed with features') }}
+          </li>
+          <li>
+            {{ _('Integrated calendar application (PIM)') }}
+          </li>
+          <li>
+            {{ _('Integrated chat client') }}
+          </li>
+          <li>
+            {{ _('Advanced Search and Filter tools') }}
+          </li>
+          <li>
+            {{ _('Inbuilt message encryption (OpenPGP, S/MIME)') }}
```

Comment:
Strictly speaking, this should be `Built-in` for "en-US" english.

Reply:
Perfect!